userguide/dailyuse/anbox.rst

```diff
@@ -14,12 +14,12 @@ Supported devices
 
 Make sure your device is supported:
 
-- Meizu Pro 5 (codename: ``turbo``, name of the boot partition: ``bootimg``)
-- Fairphone 2 (codename: ``FP2``, name of the boot partition: ``boot``)
-- OnePlus One (codename: ``bacon``, name of the boot partition: ``boot``)
-- Nexus 5 (codename: ``hammerhead``, name of the boot partition: ``boot``)
-- BQ M10 HD (codename: ``cooler``, name of the boot partition: ``boot``)
-- BQ M10 FHD (codename: ``frieza``, name of the boot partition: ``boot``)
+- Meizu Pro 5
```

Comment:
```suggestion
- Meizu PRO 5
```
As per https://en.wikipedia.org/wiki/Meizu_PRO_5 and https://www.meizu.com/en/products/pro5/spec.html